common/time.hpp

```diff
@@ -15,4 +15,12 @@ static inline Time timestamp() {
     return (Time)time.tv_sec * 1000000 + (Time)time.tv_usec;
 }
 
+static inline RJ::Time SecsToTimestamp(double secs){
```

Comment:
Is there a reason this takes in a double. You should also add a comment that this returns a u_int64 and gets rid of decimals.